From 88dd7c11b4b8a7939c7c86f914d484c88b9e41d5 Mon Sep 17 00:00:00 2001
From: milaq <micha.laqua@gmail.com>
Date: Sun, 21 Dec 2014 14:23:33 +0100
Subject: [PATCH] recovery: non-touch hacks

this is a recovery, not an OS you should use regularly
---
 screen_ui.cpp | 6 +++---
 ui.cpp        | 2 ++
 2 files changed, 5 insertions(+), 3 deletions(-)

diff --git a/screen_ui.cpp b/screen_ui.cpp
index eb3352e..68f5162 100644
--- a/screen_ui.cpp
+++ b/screen_ui.cpp
@@ -278,7 +278,7 @@ void ScreenRecoveryUI::draw_dialog()
             gr_text(2, y, text[row], 0);
             y += cy+2;
         }
-        gr_set_font("menu");
+        gr_set_font("log");
     }
 
     if (dialog_icon == ERROR) {
@@ -340,7 +340,7 @@ void ScreenRecoveryUI::draw_screen_locked()
                 --row;
                 if (row < 0) row = log_text_rows-1;
             }
-            gr_set_font("menu");
+            gr_set_font("log");
             return;
         }
 
@@ -452,7 +452,7 @@ void ScreenRecoveryUI::Init()
 
     gr_set_font("log");
     gr_font_size(&log_char_width, &log_char_height);
-    gr_set_font("menu");
+    gr_set_font("log");
     gr_font_size(&char_width, &char_height);
 
     log_text_rows = gr_fb_height() / log_char_height;
diff --git a/ui.cpp b/ui.cpp
index a1587dc..1418458 100644
--- a/ui.cpp
+++ b/ui.cpp
@@ -291,6 +291,7 @@ int RecoveryUI::input_callback(int fd, uint32_t epevents, void* data)
     }
 
     switch (ev.type) {
+#ifdef TOUCH_ENABLE
     case EV_SYN:
         self->process_syn(dev, ev.code, ev.value);
         break;
@@ -300,6 +301,7 @@ int RecoveryUI::input_callback(int fd, uint32_t epevents, void* data)
     case EV_REL:
         self->process_rel(dev, ev.code, ev.value);
         break;
+#endif
     case EV_KEY:
         self->process_key(dev, ev.code, ev.value);
         break;
-- 
2.1.1

